src/content/developers/docs/scaling/zk-rollups/index.md

```diff
@@ -11,6 +11,8 @@ You should have a good understanding of all the foundational topics and a high-l
 
 Looking for a more beginner-friendly resource? See our [introduction to layer 2](/layer-2/).
 
+You should also be familiar with ZKP technology. See our [primer on zero-knowledge proofs](/zero-knowledge-proofs/).
```

Comment:
```suggestion
You should also be familiar with zero knowledge as a concept. See our [introduction to zero-knowledge proofs](/zero-knowledge-proofs/).
```

Reply:
Maybe not as a concept... brain has switched off

Reply:
```suggestion
You should also be familiar with zero-knowledge proofs. See our [introduction to zero-knowledge proofs](/zero-knowledge-proofs/).
```

Not a strong opinion. In favor of a mixture of the two, but would prefer no acronym I think.